qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -215,12 +216,8 @@ def on_mode_left(self, mode):
             self.hide_completion.emit()
 
     def setText(self, text):
-        """Extend setText to set prefix and make sure the prompt is ok."""
-        if not text:
-            pass
-        elif text[0] in modeparsers.STARTCHARS:
-            super().set_prompt(text[0])
-        else:
+        """Extend setText to make sure the prefix is valid."""
+        if text and text[0] not in modeparsers.STARTCHARS:
```

Comment:
This still breaks things like `<Home>` :wink:

Reply:
How can I reproduce this problem? When I'm in command mode I can't even press Home.

Reply:
Or do you mean the home method ?